pandas/tests/arrays/interval/test_interval.py

```diff
@@ -45,6 +49,20 @@ def test_where_raises(self, other):
         with pytest.raises(ValueError, match=match):
             ser.where([True, False, True], other=other)
 
+    def test_overlaps(self, interval_overlaps_params):
```

Comment:
Likewise this should be moved to `pandas/tests/arrays/interval/test_ops.py` and leverage some of the existing machinery there.

Reply:
ditto as for scalar comment.